.github/workflows/dapr.yml

```diff
@@ -173,4 +181,11 @@ jobs:
             --body "${RELEASE_BODY}" \
             --prerelease true \
             ${RELEASE_ARTIFACT[*]}
+      - name: Upload Helm charts to Azure Container Registry
+        if: startswith(github.ref, 'refs/tags/v')
+        run: |
+          export HELM_EXPERIMENTAL_OCI=1
+          echo ${{ secrets.SP_PASSWD }} | helm registry login ${{ secrets.ACR_NAME }} --username ${{ secrets.SP_APP_ID }} --password-stdin
+          helm chart save ${{ env.HELM_CHART_ROOT }} ${{ secrets.ACR_NAME }}/${{ env.HELM }}/${{ env.RELEASE_NAME }}:${{ env.DAPR_VERSION }}
+          helm chart push ${{ secrets.ACR_NAME }}/${{ env.HELM }}/${{ env.RELEASE_NAME }}:${{ env.DAPR_VERSION }}
```

Comment:
I would move these three lines to makefile.
```
export HELM_EXPERIMENTAL_OCI=1
helm chart save ${{ env.HELM_CHART_ROOT }} ${{ secrets.ACR_NAME }}/${{ env.HELM }}/${{ env.RELEASE_NAME }}:${{ env.DAPR_VERSION }}
helm chart push ${{ secrets.ACR_NAME }}/${{ env.HELM }}/${{ env.RELEASE_NAME }}:${{ env.DAPR_VERSION }}
```

Reply:
Will do